README.md

```diff
@@ -509,9 +532,11 @@ Wrap this code into a new function `react` that takes a `workflowId` and an `eve
 
 Finally, pass the `react` function as a parameter to the only call of `listenForTransactions`.
 
+The app is almost ready to start, there's only one tweak to make: before writing the code to react to transactions, in order for us to observe the transactions while running a single application, the code you wrote subscribed to the ledger from it's beginning (`LedgerOffsetBoundaryValue.BEGIN`). The next time you'll run the application you'll instead _tail_ the ledger subscribing to the end of if (`LedgerOffsetBoundaryValue.END`) so that you can only observe the live interaction between two running applications.
```

Comment:
I don't quite understand what the first sentence after the `:` is trying to say.

Reply:
Hopefully better explained in https://github.com/digital-asset/ex-tutorial-nodejs/commit/31c20f8b85cf929172f994519a377111f011865f

Reply:
Improved on with https://github.com/digital-asset/ex-tutorial-nodejs/pull/7/commits/7cbd34f6f64471bf6912614e32c97740d4aff4ac